### ISSUE TITLE
"Help > Tutorial" now shows just 3 out of 5 steps 

### ISSUE THREAD

**Issue by efeprat** [OpenShot/openshot-qt#5022]
**Describa el problema**

"Help > Tutorial" used to show 5 steps. Now it just shows **Project Files**, **Video Preview**, and **Export Video**. Where are the steps for timeline and properties?

And sorry, but I can't test the last daily build (no free space enough for that).

**Pasos para reproducir el comportamiento:**

1. Choose "Help > Tutorial".
2. Follow that tutorial (click "Next" when available).

**Comportamiento esperado**

Being able to walk through its 5 usual steps.

**Detalles del sistema**
- OpenShot Version: 2.6.0-dev
- libopenshot Version:  0.2.6-dev
- Platform: Linux-5.15.0-52-generic-x86_64-with-glibc2.29
- Processor: x86_64
- Machine: x86_64
- Python version: 3.8.10
- Qt5 version: 5.12.8
- PyQt5 version: 5.14.1
- Qt Detected Languages: ['es-ES']
- LANG Environment Variable: es_ES.UTF-8
- LOCALE Environment Variable: 
- Compilación diaria: Saltar este paso

**Archivos de registro**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/f8fadhzghes02dlxw3l7tgygwk5xht8f/openshot-qt.zip) (3 KB)
- libopenshot.log: archivo demasiado pequeño o vacío (no fue cargado)

**Excepción / Rastreo de pila**
```No stacktrace found in log files```

**Comment by Colorjet3** [comment]
You really need to make an effort to free up disk space.  If you are that low that you can not install an update (by the way, 2.6.0 is old compared to all the new daily builds and should be upgraded) then you are asking for trouble not only in OpenShot, but all other applications as well, including your operating system.  Does this also mean you no longer download the operating system patches?

Any  way, what you can do for now is:
1. Close openshot.
2.  go to /home/username/.openshot_qt folder.  
3. make a backup of openshot.settings file.  
4. Delete openshot.settings file. 
5. start openshot and this should display all 5 help screens.
6. if not, then they have been possibly deleted.
7. you will need to reinstall v2.6.0

**Comment by Colorjet3** [comment]
hello @efeprat Any update on this?

**Comment by efeprat** [comment]
Sorry, @Colorjet3, I have just realized you commented on my issue.

I only intended to report it as a kind of contribution, just in case nobody noticed it before, it's not a fix I really needed.

If I'm the only affected user (I mean, not a problem in the OpenShot side, but in mine's), no further action is needed at all and I feel very sorry for making you waste your time. Thanks for your undeserved atention!
